sdk/src/main/java/com/qonversion/android/sdk/logger/Logger.java

```diff
@@ -6,4 +6,6 @@
 
     void log(String message);
```

Comment:
Let's rename this one to `release` or `releaseLog`. I think it will be more obvious in usage and code understanding